_extras/design.md

```diff
@@ -17,9 +17,10 @@ than Git itself.
 
 2.  A user on GitHub can only have one fork of a particular repo.
     This is a problem for us because an instructor may be involved in
-    several workshops, each of which has its own website repo.  Those
-    website repositories ought to be forks of this one, but since
-    GitHub doesn't allow that, we have to use `import.github.com`.
+    several workshops, each of which has its own website repo. GitHub
+    does not allow having multiple forks for the same repository so,
```

Comment:
The first clause of the sentence starting "GitHub does not.." is redundant to the preceding sentence